tests/basic/pure_test.go

```diff
@@ -5242,7 +5242,8 @@ var _ = Describe("{DisableCsiTopologyandDeletePool}", func() {
 			log.InfoD(stepLog)
 			nodeForPoolDelete = append(nodeForPoolDelete, stNodes[rand.Intn(len(stNodes))])
 			log.InfoD("Deleting the pool on the node [%v]", nodeForPoolDelete[0].Name)
-			deletePoolAndValidate(nodeSelected, fmt.Sprintf("%d", nodePool.ID))
+			err = DeletePoolAndValidate(nodeSelected, fmt.Sprintf("%d", nodePool.ID))
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Validate pool [%d] deletion in the node [%s]", nodePool.ID, nodeSelected.Name))
```

Comment:
I think `nodePool.ID` is a string not an int

Reply:
it is int32 .
```
type StoragePool struct {
	state         protoimpl.MessageState
	sizeCache     protoimpl.SizeCache
	unknownFields protoimpl.UnknownFields

	// Deprecated! Use `uuid` instead. ID pool ID
--> ID int32 `protobuf:"varint,1,opt,name=ID,proto3" json:"ID,omitempty"`
	// Cos reflects the capabilities of this drive pool
	Cos CosType `protobuf:"varint,2,opt,name=Cos,proto3,enum=openstorage.api.CosType" json:"Cos,omitempty"`
	// Medium underlying storage type
	Medium StorageMedium `protobuf:"varint,3,opt,name=Medium,proto3,enum=openstorage.api.StorageMedium" json:"Medium,omitempty"`
	// RaidLevel storage raid level
	RaidLevel string `protobuf:"bytes,4,opt,name=RaidLevel,proto3" json:"RaidLevel,omitempty"`
	// TotalSize of the pool
	TotalSize uint64 `protobuf:"varint,7,opt,name=TotalSize,proto3" json:"TotalSize,omitempty"`
	// Used size of the pool
	Used uint64 `protobuf:"varint,8,opt,name=Used,proto3" json:"Used,omitempty"`
	// Labels is a list of user defined name-value pairs
	Labels map[string]string `protobuf:"bytes,9,rep,name=labels,proto3" json:"labels,omitempty" protobuf_key:"bytes,1,opt,name=key,proto3" protobuf_val:"bytes,2,opt,name=value,proto3"`
	// UUID is the unique identifier for a storage pool
	Uuid string `protobuf:"bytes,10,opt,name=uuid,proto3" json:"uuid,omitempty"`
	// LastOperation is the most recent operation being performed on a storage pool
	LastOperation *StoragePoolOperation `protobuf:"bytes,11,opt,name=last_operation,json=lastOperation,proto3" json:"last_operation,omitempty"`
}
```